src/IOT/lab1/Crank1.java

```diff
@@ -0,0 +1,33 @@
+package IOT.lab1;
+
+public record Crank1(int maxLiftingWeight, String producer, String colour, int mass, int priceInUah, String size, String typeOfCrank) {
```

Comment:
If you've already written a field name in a more specific way (priceInUah instead of price - which is actually really good!), try sticking with this convention; massInKilograms, maxLiftingWeightInTons etc.